Plugins/DataSink/Lcd/LcdConfig.cs

```diff
@@ -15,6 +15,6 @@ public class LcdConfig : ConfigLoader<LcdConfig>, IConfigSaver
         /// <summary>
         /// Format to display a timestamp
         /// </summary>
-        public string TimestampFormat = "dd.MM.yyyy  HH:mm:ss";
+        public string TimestampFormat = "dd.MM.yyyy HH:mm:ss";

```

Comment:
This space was placed with reason there. The LCD has a width of 20 digits. In this mask the date is left bound, the time is right bound.

Reply:
I thought it was a typi, will change it back :+1: